internal/scorecard/alpha/scorecard.go

```diff
@@ -165,12 +163,7 @@ func (r FakeTestRunner) RunTest(ctx context.Context, test Test) (result *v1alpha
 }
 
 func ConfigDocLink() string {
-	if strings.HasSuffix(version.Version, "+git") {
-		return "https://github.com/operator-framework/operator-sdk/blob/master/doc/test-framework/scorecard.md"
-	}
-	return fmt.Sprintf(
-		"https://github.com/operator-framework/operator-sdk/blob/%s/doc/test-framework/scorecard.md",
-		version.Version)
+	return "https://sdk.operatorframework.io/docs/scorecard/"
```

Comment:
@jmccormick2001 are you ok with this change?

Reply:
fine with me